react/features/base/react/components/native/HeaderLabel.js

```diff
@@ -0,0 +1,53 @@
+// @flow
+
+import React, { Component } from 'react';
+import { Text } from 'react-native';
+
+import { translate } from '../../../i18n';
+
+import styles from './styles';
+
+/**
+ * The type of the React {@code Component} props of {@link HeaderLabel}
+ */
+type Props = {
+
+    /**
+     * The i18n key of the label to be rendered.
+     */
+    labelKey: string,
+
+    /**
+     * An external style object passed to the component.
+     */
+    style?: Object,
```

Comment:
I don't think this prop is being used. Did you intend for it to be used?

Reply:
Right, I think that was some legacy stuff left there. Thanks!